blobfuse2-code-coverage.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
           stressParallel: 3
         Ubuntu-20:
           imageName: 'ubuntu-20.04'
-          containerName: 'test-cnt-coverage'
+          containerName: 'test-cnt-ubn-20'
```

Comment:
This was explicitly done to avoid same container being used by both nightly and coverage.

Reply:
the likelihood of them being run at the same time is kinda low right?

Reply:
Code coverage for now is on demand basis but once we make it run periodically then likelihood might increase. Any specific reason you wanted to change the name here ?

Reply:
mostly cause it runs container sas tests now

Reply:
also why do we allow the ubn18 container to overlap

Reply:
it also makes managing the SASs hard since we need one per container

Reply:
I think we should minimize maintaining SASs and we can just rerun in the odd case